platform/generic/rules.mk

```diff
@@ -3,7 +3,7 @@ include $(PLATFORM_PATH)/onie-image.mk
 
 SONIC_ALL += $(DOCKER_DATABASE) \
          $(DOCKER_LLDP_SV2) \
```

Comment:
>DOCKER_LLDP_SV2 [](start = 11, length = 15)

Do you want to fix docker-lldp-sv2 in the same PR?

Reply:
I was going to do each container in separate PRs.